src/main/resources/user/walkthrough/Traversing.md

```diff
@@ -49,7 +49,7 @@ Here are the results:
 ${GET_ADJACENT_IDS_SIMPLE_RESULT}
 ```
 
-For more details and for more detailed examples of the GetAdjacentIds operation please see [GetAdjacentIds](operations/getadjacentids.md)
+For more details and for more detailed examples of the GetAdjacentIds operation please see [GetAdjacentIds](../operations/getadjacentids.md)
```

Comment:
Does `..` work once converted to HTML?

Reply:
I checked and it does work. It get converted to a full (non-relative) HTML path to the page. In HTML I think using `..` may or may not work depending on the server, which probably explains why it's done this way.
This type of path is also used in Intro.md, TheBasics.md, Filtering.md and other pages.